pkg/controller/workload/common/fake_helpers.go

```diff
@@ -42,28 +44,27 @@ func CreateFakeService(name, ip, waypoint string, optional ...interface{}) *work
 		Waypoint: w,
 	}
 
-	if len(optional) > 0 {
-		if lbPolicy, ok := optional[0].(*workloadapi.LoadBalancing); ok {
-			service.Ports = []*workloadapi.Port{
-				{
-					ServicePort: 80,
-					TargetPort:  8080,
-				},
-				{
-					ServicePort: 81,
-					TargetPort:  8180,
-				},
-				{
-					ServicePort: 82,
-					TargetPort:  82,
-				},
-			}
-			service.LoadBalancing = lbPolicy
+	if lbPolicy != nil {
+		service.Ports = []*workloadapi.Port{
+			{
+				ServicePort: 80,
+				TargetPort:  8080,
+			},
+			{
+				ServicePort: 81,
+				TargetPort:  8180,
+			},
+			{
+				ServicePort: 82,
+				TargetPort:  82,
+			},
 		}
+		service.LoadBalancing = lbPolicy[0]
```

Comment:
since here only one policy is needed, why use variable param?

Reply:
In waypoint_cache_test.go the func is called like this: `svc1 := common.CreateFakeService("svc1", "10.240.10.1", "")` so I used vardiac func, If I am make it fixed then I will have to pass `nil` during func call. will that be okay?